app/controllers/albums

```diff
@@ -0,0 +1,27 @@
+const rp = require('request-promise');
+const typicodePath = 'https://jsonplaceholder.typicode.com';
```

Comment:
When declaring several `const`, you don't need to type `const` each time. You can do the following:
```
const rp = require('request-promise'),
    typicodePath = 'https://jsonplaceholder.typicode.com';
```